one/src/oauth/one.clj

```diff
@@ -65,7 +65,8 @@
 (def OAuthAuthorization
   "Valid attributes and corresponding values allowed in the unsigned OAuth
   Authorization header."
-  {(s/optional-key "oauth_token") s/Str
+  {(s/optional-key "oauth_callback") s/Str
```

Comment:
This is marked as optional because it's not required in _every_ signed request (although it is required in the request token request).

https://dev.twitter.com/oauth/reference/post/oauth/request_token

![twitter-docs](https://cloud.githubusercontent.com/assets/18374/13793964/77c2e724-eaf1-11e5-9a15-20880e69f4d8.png)